src/common/utility_win.cpp

```diff
@@ -142,6 +154,16 @@ QVariant Utility::registryGetKeyValue(HKEY hRootKey, const QString &subKey, cons
             }
             break;
         }
+        case REG_BINARY: {
+            BYTE buf[255];
+            DWORD bufSize = sizeof(buf);
+            Q_ASSERT(sizeInBytes <= bufSize);
+            result = RegQueryValueEx(hKey, reinterpret_cast<LPCWSTR>(valueName.utf16()), 0, &type, reinterpret_cast<LPBYTE>(&buf), &sizeInBytes);
+            if (result == ERROR_SUCCESS) {
+                value = buf[12];
+            }
```

Comment:
I haven't checked this, but it looks very hacky. But it's probably the way things are done with the Windows API.